object-construction-checker/tests/mustcall/MustCallChoiceImpl.java

```diff
@@ -0,0 +1,19 @@
+//import org.checkerframework.checker.mustcall.qual.*;
```

Comment:
Why is this all commented out?

Reply:
I uncommented it. the unexpected error that I got is documented in the test.